src/Lykke.Service.PayInvoice.Repositories/MerchantSettingsEntity.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Lykke.AzureStorage.Tables;
+using Lykke.AzureStorage.Tables.Entity.Annotation;
+using Lykke.AzureStorage.Tables.Entity.ValueTypesMerging;
+
+namespace Lykke.Service.PayInvoice.Repositories
+{
+    [ValueTypeMergingStrategy(ValueTypeMergingStrategy.UpdateIfDirty)]
+    public class MerchantSettingEntity : AzureTableEntity
+    {
+        private string _baseAsset;
+        
+        public MerchantSettingEntity()
+        {
+        }
+
+        public MerchantSettingEntity(string partitionKey, string rowKey)
+        {
+            PartitionKey = partitionKey;
+            RowKey = rowKey;
+        }
+        
+        public string BaseAsset
```

Comment:
This approach is needed only for value types. For strings it is overhead. Please remove.

Reply:
Fixed https://github.com/LykkeCity/Lykke.Service.PayInvoice/pull/13